examples/highschool_physics_example.ped

```diff
@@ -0,0 +1,15 @@
+// run with: 
+// $ stack run pedant compile examples/highschool_physics_example.ped
+unit verticalMeters seconds kilograms jules newtons
+mass = 10 kilograms
+initial_height = 10 verticalMeters
+speed_upwards = 5 verticalMeters / (1 seconds)
+duration_of_movement = 10 seconds
+gravity_constant = 9.8 verticalMeters / (1 seconds2)
+upwards_movement  = speed_upwards * duration_of_movement
+final_vertical_height  = initial_height + upwards_movement
+joule_conversion = 1 jules / (1 kilograms * 1 verticalMeters2 * 1 seconds-2)
```

Comment:
The fact that you have this gives me a bigger reason to add type aliases, say for instance `alias jules = seconds2 kilograms-1 verticalMeters-2`, and the checker infers the rest.

Reply:
Might be worth having as a feature eventually, but unclear whether to prioritize it.